test/bin/manage_prometheus.sh

```diff
@@ -45,7 +45,7 @@ action_start() {
     podman run -d --rm --name prometheus \
         -p 9092:9090 \
         -v "${PROMETHEUS_DIR}:/etc/prometheus:Z" \
-        docker.io/prom/prometheus \
+        quay.io/prometheus/prometheus \
```

Comment:
Good catch, I had trouble with a `registry.redhat.io` image and somehow didnt try quay.

Reply:
Thanks, change it also for better performance in Jenkins pipelines.